### DISCUSSION TITLE
nvm fixed

### DISCUSSION THREAD

**Discussion by TailsGamingYT** [KadeArchive/Kade-Engine#2412] [Q&A]
im kinda new t git hub, dont know how to remove things i figured out.

**Comment by tahiadz1962** [comment]
hi bud its me urjust2ez4me
remember me?
we used to chill out with tannermanner

well my discord got hacked
=(

**Reply by TailsGamingYT** [reply]
oh, yeah, Hi!, im so sorry about ur account getting hacked, and sorry for my late reply too, i never get notified for anything usually lol,
